sql/core/src/main/scala/org/apache/spark/sql/execution/debug/package.scala

```diff
@@ -156,8 +156,8 @@ package object debug {
     def typeCheck(data: Any, schema: DataType): Unit = (data, schema) match {
       case (null, _) =>
 
-      case (row: InternalRow, StructType(fields)) =>
-        row.toSeq.zip(fields.map(_.dataType)).foreach { case(d, t) => typeCheck(d, t) }
+      case (row: InternalRow, s: StructType) =>
```

Comment:
@marmbrus is this debug package still useful given the massive internal refactoring from tungsten lately?